src/app.rs

```diff
@@ -287,11 +312,19 @@ impl<T: Send + 'static, V: View<T> + 'static> App<T, V> {
             };
 
             root_pod.paint(&mut paint_cx);
-            execute!(stdout(), BeginSynchronizedUpdate)?;
+
+            #[cfg(not(test))]
+            {
+                execute!(stdout(), BeginSynchronizedUpdate)?;
+                self.terminal.flush()?;
```

Comment:
Shouldn't this still be enabled? (as well as the flush swap_buffers and flush below)?

(So just above the `execute!` macros)

Reply:
I somehow missed the swap_buffers(), but the flush() is covered in line 327.

Reply:
You're right, missed that.

I pushed this and some smaller fixes to get it ready to merge, hope that's fine for you.